<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierWindow</class>
 <widget class="QWidget" name="CashierWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Кассир</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSale">
      <attribute name="title">
       <string>Оформление продажи</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Товары</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSearch">
           <property name="placeholderText">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="twProducts"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0,0">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Итог</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="twCart"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Цена без скидки</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lCostWithoutDiscount">
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Скидка</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="dsbDiscount">
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
             <property name="prefix">
              <string/>
             </property>
             <property name="suffix">
              <string>%</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Итог</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lTotal">
             <property name="font">
              <font>
               <pointsize>16</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbSave">
           <property name="text">
            <string>Пробить чек</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSales">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
